src/webextension/list/actions.js

```diff
@@ -33,6 +36,37 @@ export const HIDE_MODAL = Symbol("HIDE_MODAL");
 // other (i.e. FOO_STARTING and FOO_COMPLETED).
 let nextActionId = 0;
 
+export function accountStatus() {
```

Comment:
As above, I'd call this `getAccountDetails` or `getAccount`; actions are verbs, and "status" was confusing to me.